docs-src/docs/getting-started/quickstart.mdx

```diff
@@ -48,6 +48,11 @@ export KOGNIC_CREDENTIALS=~/path/to/credentials.json
 
 To verify that they are correct you can simply list projects in Python:
 
+::::note
+Since kognic-io v2.4.0 you can provide a workspace id that will be used for writes.
+For users with multiple workspaces this will be mandatory. [See](../kognic_apis.md#specifying-a-workspace) 
```

Comment:
```suggestion
For users with multiple workspaces this will be mandatory. [See this page for instructions.](../kognic_apis.md#specifying-a-workspace) 
```